Source/Fuse.Controls.ScrollView/Scroller.uno

```diff
@@ -198,9 +200,26 @@ namespace Fuse.Gestures
 		{
 			get { return _scrollable == null ? GesturePriority.Lower : _scrollable.GesturePriority; }
 		}
+
+		bool _pressed;
+		void StartInvalidateVisual()
+		{
+			if (!_pressed)
+				UpdateManager.AddAction(_scrollable.InvalidateVisual);
```

Comment:
Hmm, don't we have a way to just prevent caching instead? Forcing invalidation could actually hurt performance if the user is idling.